doc/cloud/cloud-welcome.rst

```diff
@@ -0,0 +1,39 @@
+Welcome
+============
+
+What is KNoT Cloud? 
+-------------------
+
+KNoT Cloud is a cloud solution developed by CESAR that aims to connect IoT devices easily and securely to a minimal messaging and management infrastructure. It supports WebSockets protocol to communicate with the core IoT services and provides APIs that lets you managing users and collecting historical data sent by the devices. 
+
+This solution makes it easy to build an IoT application that gathers, process, analyze and act on data generated by connected devices. A software development kit (`SDK <cloud-api.html>`_) is available to facilitate the application development by providing an easy to use API to the main services (device messaging, data storage and authenticator).
+
+----------------------------------------------------------------
+
+Features
+--------
+
+Distributed Messaging
+'''''''''''''''''''''''''''
+   Built on top of Meshblu components, the KNoT Cloud can process the messages across lightweight microservices to deliver a scalable system.
+
+Data Storage
+''''''''''''
+   The data sent by IoT devices are stored on a non-relational database.
+
+Incorporated Protocol
+'''''''''''''''''''''
+Some KNoT protocol characteristics were brought to the cloud protocol adapters to build a reliable model layer of device representation and supports scenarios in which devices can connect directly to the cloud through the network.
+
+----------------------------------------------------------------
+
+Architecture
+------------
+
+.. image:: ../../_static/cloud_overview_architecture.png
+   :alt: alternate text
```

Comment:
Give an appropriate name for ":alt:"
- alt attribute specifies an alternate text for an image, if the image cannot be displayed.

Reply:
Done.